go/teams/chain.go

```diff
@@ -2196,10 +2208,21 @@ func (t *teamSigchainPlayer) updateInvites(stateToUpdate *TeamSigChainState, add
 	}
 }
 
-func (t *teamSigchainPlayer) completeInvites(stateToUpdate *TeamSigChainState, completed map[keybase1.TeamInviteID]keybase1.UserVersionPercentForm) {
+func (t *teamSigchainPlayer) completeInvites(stateToUpdate *TeamSigChainState, completed map[keybase1.TeamInviteID]keybase1.UserVersionPercentForm) error {
 	for id := range completed {
+		invite, ok := stateToUpdate.inner.ActiveInvites[id]
+		if !ok {
+			// TODO: That's likely an error, but we've allowed this for so long
+			// that we have to prove that no team has done this in order to
+			// change this.
```

Comment:
I mostly agree, but would drop the `TODO`.

Reply:
How about you make a ticket and put a number in there instead of TODO? Eventually we should fix this and potentially make a list of exceptions to whitelist. Probably not a big priority though.

Reply:
this is only one example of the player being loose. we can collect potential fixes in there but it's a big effort to go through this cycle for each one of them
- check if there are any teams abusing that
   - if there are many, maybe don't change that
   - if there are few, make list with exceptions
- add tests cases
- change code